Philips.CodeAnalysis.Test/Maintainability/Maintainability/DontLockNewObjectAnalyzerTest.cs

```diff
@@ -1,11 +1,13 @@
 ﻿// © 2023 Koninklijke Philips N.V. See License.md in the project root for license information.
 using System;
+using System.Reflection.Metadata;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Philips.CodeAnalysis.Common;
 using Philips.CodeAnalysis.MaintainabilityAnalyzers.Maintainability;
 using Philips.CodeAnalysis.Test.Helpers;
 using Philips.CodeAnalysis.Test.Verifiers;
+using static System.Runtime.InteropServices.JavaScript.JSType;
```

Comment:
Please remove this unused using

Reply:
nice catch. will take care of in round 2.